dimod/discrete/discrete_quadratic_model.py

```diff
@@ -537,6 +537,10 @@ def from_numpy_vectors(cls, case_starts, linear_biases, quadratic,
 
         return obj
 
+    def get_cases(self, v):
```

Comment:
do we need to add this method to dqm.rst?

Reply:
Also the changelog I suppose